internal/testutil/testutil.go

```diff
@@ -141,7 +159,14 @@ func NewManager(t *testing.T) *Manager {
 			panic(err)
 		}
 	})
-	m.DownstreamRestConfig, err = downstreamEnv.Start()
+	for i := 0; i < 2; i++ {
```

Comment:
Sometimes tests flake because this fails to start. 3 retries seems to be enough.